src/app/tests/suites/certification/Test_TC_CC_1_1.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 21.1.1. [TC-CC-1.1] Global attributes with server as DUT
+
+config:
+    cluster: "Color Control"
+    endpoint: 1
+
+tests:
+    - label: "read the global attribute: ClusterRevision"
+      disabled: true
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 4
+
+    - label:
+          "write the default values to mandatory global attribute:
+          ClusterRevision"
+      disabled: true
```

Comment:
You can remove disabled for all `writeAttribute` action now.

Reply:
Vivien , in all the 20 odd yamls that we have the write attributes are disabled for now . Let me verify each of them and put up a separate PR to enable them all .